frontend/src/pages/stats/index.tsx

```
@@ -51,7 +51,7 @@ class Stats extends PureComponent {
           </div>
           <NetworkConsumer>
             {({ currentNearNetwork }) =>
-              currentNearNetwork.name === "mainnet" ? (
+              currentNearNetwork?.name === "mainnet" ? (
```

Those `?`s seem a little excess. We probably should make a hook which encapsulate network existence or make a default network

---

My only concern would be that we may potentially hide some bugs lurking due to the fallback always hiding non-set value, but it seems it should not be the case with the current network

---

I'll make encapsulate it in a hook in a PR soon, it'll throw error if there is no current network.